rpc/documentation/api.md

```diff
@@ -1615,6 +1681,53 @@ wallet's relevant transactions contained therein.
 
 ___
 
+#### `TicketDetails`
+
+The `TicketDetails` message is included in responses to report transactions
+relevant to the wallet.  The message includes details such as which previous
+wallet inputs are spent by this transaction, whether each output is controlled
+by the wallet or not, the total fee (if calculable), and the earlist time the
+transaction was seen.
```

Comment:
This description is a copy pasta of TransactionDetails' description and needs to be redone.